src/rules/no-property-change-update.ts

```diff
@@ -52,8 +52,7 @@ const rule: Rule.RuleModule = {
       }
 
       const props = getPropertyMap(node);
-
-      if (props) {
+      if (props.size > 0) {
```

Comment:
@43081j Note that `props` is a `Map` so it is always truthy. Which is why I removed it previously. I believe this update adds your intentions? Please let me know if you want it changed to something else.

Reply:
i had a look through and i suspect i assumed i made `getPropertyMap` return null when there isn't one. but actually it just returns the empty map.

you can have an empty properties map so lets just drop the if